rules/autoconsent/sibbo.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "sibbo",
+  "prehideSelectors": [".sibbo-layout, .sibbo-panel"],
+  "detectCmp": [{ "exists": "sibbo-cmp-layout" }],
+  "detectPopup": [{ "visible": ".sibbo-layout" }],
+  "optIn": [{ "click": "#acceptAllMain" }],
```

Comment:
I see that this button has some data attributes, that might be more stable. How about `.sibbo-layout [data-accept-all]`? Would that work?

Reply:
yes it does work.